src/pages/_document.tsx

```diff
@@ -73,18 +73,17 @@ const getCspContent = (context) => {
         ' '
       )} ${ANALYTICS_CSP.alpha.img.join(' ')}; 
       media-src 'self';
-      script-src 'unsafe-eval' 'self' ${cspInlineScriptHash} ${ANALYTICS_CSP.alpha.script.join(
+      script-src 'unsafe-eval' 'self' *.shortbread.aws.dev ${cspInlineScriptHash} ${ANALYTICS_CSP.alpha.script.join(
```

Comment:
Should we add these to the `ANALYTICS_CSP` object above? I think  you could add a new `script` property in `ANALYTICS_CSP.all`.